app/code/community/Studioforty9/Recaptcha/Block/Explicit.php

```diff
@@ -169,9 +167,19 @@ public function getRecaptchaScript($id)
      */
     public function getRecaptchaId()
     {
-        return uniqid();
+        return $this->recaptchaId;
     }
 
+    public function getTemplate()
+    {
+        if ($this->_getHelper()->getVersion() === 3) {
+            return 'studioforty9/recaptcha/v3.phtml';
+        }
+
+        return 'studioforty9/recaptcha/explicit.phtml';
```

Comment:
Non mi dispiacerebbe rinominare questo template in "v2" per coerenza, ma poi ci potrebbero essere dei problemi di BC break se qualcuno overridava il vecchio template...

Reply:
Si infatti è il motivo per cui l'ho lasciato così.